src/torchcodec/_core/custom_ops.cpp

```diff
@@ -381,6 +384,38 @@ OpsAudioFramesOutput get_frames_by_pts_in_range_audio(
   return makeOpsAudioFramesOutput(result);
 }
 
+at::Tensor wrapEncoderPointerToTensor(std::unique_ptr<Encoder> uniqueEncoder) {
+  Encoder* encoder = uniqueEncoder.release();
+
+  auto deleter = [encoder](void*) { delete encoder; };
+  at::Tensor tensor =
+      at::from_blob(encoder, {sizeof(Encoder)}, deleter, {at::kLong});
```

Comment:
It should be `sizeof(Encoder*)` since we want space just to store the pointer, not the entire object.